### PR TITLE
Add Zephyr Squad documentation pages

### PR DESCRIPTION
**Pull request by pelizza**
Zephyr Squad (2.0) API docs.

### CODE REVIEW

File: products/Zephyr Squad/Introduction.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+This is the **Zephyr Squad Cloud** API. If you are looking for the Server API, please refer to the documentation for more information.
+This API uses the OpenAPI specification. If you want to generate clients for different languages, please use the link above to download the API specification and use it with your preferred tools.
+For feature requests or general support, please head to our support site.
```
Comment (frankkilcommins) [inline products/Zephyr Squad/Introduction.md:3]: would you like to link to the support site here?

File: products/Zephyr Squad/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+This file contains metadata used to specify what APIs to link to a product and what section order documentation will follow.
+
+## Content Metadata
+
+| Order | Name | Type | ContentURL |
+| ----- | ---- | ---- | ---------- |
+| 0 | Introduction | `markdown` | ./Introduction.md |
+| 1 | Authentication | `markdown` | ./Authentication.md |
+| 2 | Zephyr Squad API | `apiURL` | https://app.swaggerhub.com/apis/smartbear/zephyr-squad_cloud_api/2 |
```
Comment (frankkilcommins) [inline products/Zephyr Squad/README.md:9]: FYI - this URL will change but I can do that once the API is published in to our `smartbear-public` org in SwaggerHub

File: products/Zephyr Squad/Authentication.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+The API authentication is based on JSON Web Token (JWT), which is a key for accessing the API. Authenticating using JWT requires the following steps:
+
+* Generate a key
+* Make authenticated requests
+
+## Generate a Key
+
+For accessing the API, you must generate an access key in Jira. To generate an access token, click on your profile picture at the page bottom left, and choose the option “Zephyr Squad API keys". For more information, please check out the documentation.
```
Comment (frankkilcommins) [inline products/Zephyr Squad/Authentication.md:8]: profile picture will not be on the portal page, so this should be explained that they need to navigate to Atlassian click on their profile.

Comment (frankkilcommins) [reply products/Zephyr Squad/Authentication.md:8]: should 'check out the documentation' have a link?

File: products/Zephyr Squad/Introduction.md (hunk 1 above)
Comment (TimotheeVille) [reply products/Zephyr Squad/Introduction.md:3]: Hey @frankkilcommins, yes the link should be added but we don't have it yet. I'm currently looking with the Product Owner what should be used here

**Pull request merged**